server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const port = 3000;
```

Comment:
Consider using environment variables for the port number. This will make your application more flexible and secure. You can use the dotenv package to load environment variables. [important]